src/settings_gui.cpp

```diff
@@ -217,7 +218,8 @@ struct GameOptionsWindow : Window {
 
 			case WID_GO_LANG_DROPDOWN: { // Setup interface language dropdown
 				for (uint i = 0; i < _languages.size(); i++) {
-					auto item = new DropDownListParamStringItem(STR_JUST_RAW_STRING, i, false);
+					if (IsReleasedVersion() && !_languages[i].IsReasonablyFinished()) continue;
+					auto item = new DropDownListParamStringItem(IsReleasedVersion() || _languages[i].missing < _settings_client.gui.missing_strings_threshold ? STR_JUST_RAW_STRING : STR_GAME_OPTIONS_LANGUAGE_PERCENTAGE, i, false);
```

Comment:
This conditional is complex/long enough that I think it should probably be broken into its own variable